src/wp-content/themes/twentysixteen/js/functions.js

```diff
@@ -4,210 +4,290 @@
+				var next = el.nextSibling;
+				if ( next && matches( next, '.children, .sub-menu' ) ) {
+					next.classList.toggle( 'toggled-on' );
+				}
 
-			// jscs:disable
-			_this.attr( 'aria-expanded', _this.attr( 'aria-expanded' ) === 'false' ? 'true' : 'false' );
-			// jscs:enable
-			screenReaderSpan.text( screenReaderSpan.text() === screenReaderText.expand ? screenReaderText.collapse : screenReaderText.expand );
+				el.setAttribute(
+					'aria-expanded',
+					el.getAttribute( 'aria-expanded' ) === 'false' ? 'true' : 'false'
+				);
+				screenReaderSpan.textContent =
+					screenReaderSpan.textContent === screenReaderText.expand ? screenReaderText.collapse: screenReaderText.expand;
+			} );
 		} );
 	}
-	initMainNavigation( $( '.main-navigation' ) );
+	initMainNavigation( document.querySelector( '.main-navigation' ) );
 
-	masthead         = $( '#masthead' );
-	menuToggle       = masthead.find( '#menu-toggle' );
-	siteHeaderMenu   = masthead.find( '#site-header-menu' );
-	siteNavigation   = masthead.find( '#site-navigation' );
-	socialNavigation = masthead.find( '#social-navigation' );
+	masthead = document.querySelector( '#masthead' );
+	menuToggle = masthead && masthead.querySelector( '#menu-toggle' );
+	siteHeaderMenu = masthead && masthead.querySelector( '#site-header-menu' );
+	siteNavigation = masthead && masthead.querySelector( '#site-navigation' );
+	socialNavigation = masthead && masthead.querySelector( '#social-navigation' );
 
 	// Enable menuToggle.
-	( function() {
-
+	( function () {
 		// Return early if menuToggle is missing.
-		if ( ! menuToggle.length ) {
+		if ( ! menuToggle ) {
 			return;
 		}
 
+		var controls = [ menuToggle, siteNavigation, socialNavigation ].filter( Boolean );
+
 		// Add an initial values for the attribute.
-		menuToggle.add( siteNavigation ).add( socialNavigation ).attr( 'aria-expanded', 'false' );
+		controls.forEach( function ( el ) {
+			el.setAttribute( 'aria-expanded', 'false' );
+		} );
 
-		menuToggle.on( 'click.twentysixteen', function() {
-			$( this ).add( siteHeaderMenu ).toggleClass( 'toggled-on' );
+		menuToggle.addEventListener( 'click', function () {
+			menuToggle.classList.toggle( 'toggled-on' );
+			siteHeaderMenu.classList.toggle( 'toggled-on' );
 
-			// jscs:disable
-			$( this ).add( siteNavigation ).add( socialNavigation ).attr( 'aria-expanded', $( this ).add( siteNavigation ).add( socialNavigation ).attr( 'aria-expanded' ) === 'false' ? 'true' : 'false' );
-			// jscs:enable
+			var newVal = menuToggle.getAttribute( 'aria-expanded' ) === 'false' ? 'true' : 'false';
+			controls.forEach( function ( el ) {
+				el.setAttribute( 'aria-expanded', newVal );
+			} );
 		} );
 	} )();
 
 	// Fix sub-menus for touch devices and better focus for hidden submenu items for accessibility.
-	( function() {
-		if ( ! siteNavigation.length || ! siteNavigation.children().length ) {
+	( function () {
+		if ( ! siteNavigation || siteNavigation.children.length === 0 ) {
```

Comment:
Bit of a nitpick, but I think we could keep this simpler without the `0` comparison
```suggestion
		if ( ! siteNavigation || ! siteNavigation.children.length ) {
```